changelogs/fragments/100-fix-update_failture_action-keyerror-in-docker_swarm_service.yaml

```
@@ -0,0 +1,3 @@
+bugfixes:
+  - ``docker_swarm_service`` - fix KeyError on caused by reference to deprecated
+    option ``update_failure_action`` (https://github.com/ansible-collections/community.docker/pull/100).
```

```suggestion
  - "docker_swarm_service - fix ``KeyError`` caused by reference to deprecated
    option ``update_failure_action`` (https://github.com/ansible-collections/community.docker/pull/100)."
```
(The quotes should fix whatever is caused that YAML syntax error.)

---

Thanks. Fixed.